tests/acceptance/a11y-audit-test.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 import { module, test } from 'qunit';
 import startApp from '../../tests/helpers/start-app';
-import a11yAudit from 'ember-a11y-testing/test-support/audit';
+import { a11yAudit } from 'ember-a11y-testing/test-support/audit';
```

Comment:
I could not figure out how to simulate the availability of the `enableA11yAudit` param, thus have not written any test for the newly added `a11yAuditIf`. Any pointer might help me to write tests for `a11yAuditIf`.